openmetadata-docs/content/connectors/pipeline/dagster/airflow.md

```diff
@@ -52,8 +52,12 @@ source:
   serviceConnection:
     config:
       type: Dagster
-      hostPort: http://localhost:8080
-      numberOfStatus: 10
+        # For Local Connection
+        # hostPort: http://locahost:3000/
+        
+        # For Cloud Connection
+        host: "https://<yourorghere>.dagster.cloud/prod"
```

Comment:
@NiharDoshi99 just curious can `/prod` change or is it always the case? ?

Reply:
it is always the case.